src/main/java/pl/lukakan/showstracker/cast/person/PersonService.java

```diff
@@ -28,6 +34,12 @@ public PersonService(PersonRepository personRepository) {
         return Arrays.asList(Function.values());
     }
 
+    public Set<Person> findAllPersonInFunction(String functionName) {
+        Function function = Function.getFunctionByName(functionName);
+        List<Role> roles = roleRepository.findAllByFunctionType(function);
+        return roles.stream().map(role -> role.getPerson()).collect(Collectors.toSet());
```

Comment:
Nie dałoby się tego zrobić za pomocą "lepszego" zapytnia do repository? :P

Reply:
best customowego query to chyba średnio... chyba ze ty masz jakis pomysł?

Reply:
przypominam sie z tematem :)